EssentialEstablishmentGenerator/NPCGeneration/SetupHistory.js

```diff
@@ -1,214 +1,138 @@
 
-setup.createHistory = function (town, npc) {
-  console.log('creating history for ' + npc.name + '...')
-  let wealthModifier
+setup.birthplaceTable = [
```

Comment:
I'm not sure that I agree with this implementation; I think that expressing the likelihood through relative values rather than absolutes would make it clearer which are weighted higher; i.e.
```
[50, 'at home'],
[5, 'in the home of a friend'],
[9, 'in the home of a midwife'],
[1, 'on a wagon']
etc etc. 
```
This would keep it consistent with how weighted probabilities are handled (as used in `setup.familyUnits` below), and also ensures that there is no implied limit of it being limited to a 1-100 roll. It also allows easier testing of any bugs that may crop up by pumping up the relevant probability (not that there are any in this!).